### PR TITLE
Convert Fellowship to a completely async course

### PR DESCRIPTION
**Pull request by ChaseC99**
This PR moves the Fellowship away from a weekly model and let's students to complete the material at their own pace.

### CODE REVIEW

**Comment by ChaseC99** [comment]
TODO:
- [ ] Change all references of "this week" (and similar language) to "this unit"
- [ ] Unit 2
  - [ ] Figure out what to do with the terminal screenshot. Should it still be posted in Discord?
- [ ] Unit 5
  - [ ] Make #resume-review Discord channel public and have people post there, instead of signing up on the google form
  - [ ] Create #networking Discord channel for people to post share their LinkedIns, instead of posting in a fellowship thread
  - [ ] Update the assignment accordingly
- [ ] Unit 10
  - [ ] Should we get rid of the idea of "fellowship" tags and just use "Good first task"?

File: docs/unit10.mdx [hunk 1]
```diff
@@ -23,7 +25,7 @@ Most open source projects have their code on GitHub.
 This makes it extremely easy for people to contribute to them. 
 
 :::note Fun Fact
-ICSSC projects are all open sourced. Over 50 students have contributed to the projects!
+ICSSC projects are all open sourced. Over 100 students have contributed to the projects!
```
Comment (ChaseC99) [inline docs/unit10.mdx:28]: It's actually 95 but it will be over 100 soon lol

Comment (Voark) [reply docs/unit10.mdx:28]: Fair enough 😄

File: docs/unit2.mdx [hunk 2]
```diff
@@ -198,7 +198,7 @@ Call the first one `vim.txt` and the second one `nano.txt`.
 ### Deliverables
 
 **Required Tasks**
- - Follow the directions above to practive using the terminal.
+ - Follow the directions above to practice using the terminal.
```
Comment (ChaseC99) [inline docs/unit2.mdx:201]: oh good catch!

**Comment by ChaseC99** [comment]
@ramanxg any feedback or are we good to go?

**Review by ramanxg — APPROVED** [review]
Sorry about the lateness, but it looks good to me!

**Pull request merged**